tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -259,6 +262,11 @@ function kbs_k8s_deploy() {
 EOF
 		export DEPLOYMENT_DIR=custom_pccs
 	fi
+
+	if [ "${KBS_INGRESS}" = "nodeport" ]; then
+		export DEPLOYMENT_DIR=nodeport
```

Comment:
Is it possible to have qemu-tdx with nodeport? If so which DEPLOYMENT_DIR is expected? (shouldn't we set it only when not set already by user or in code?)

Reply:
@ldoktor according to the documentation you need to export that variable

Reply:
Sure, but currently there are 2 places where we export them and they are not mutually exclusive. So it might happen that this branch overrides the `custom_pccs`. I don't know whether it might be a problem and which one should get precedence. (also whether it's likely that the user might want to override it manually - which I don't think so)